server/lib/models/account.go

```diff
@@ -37,27 +38,20 @@ func NewAccount(dbif Dbif, id *string, typ proto.Model_Account_Type, user string
 		log.Printf("a = %v", a)
 		created = true
 	} else {
-		tmp, err := strconv.ParseUint(*id, ACCOUNT_ID_BASE, 64)
-		if err != nil {
-			return nil, false, err
-		}
-		a.Id = proto.UUID(tmp)
+		a.Id = *id
 	}
 	if err := dbif.SelectOne(a, dbm.Stmt("select * from %s.accounts where id=$1"), a.Id); err != nil {
 		return nil, false, err
 	}
+	a.pmap = make(map[proto.UUID]*Persona)
 	return a, created, nil
 }
 
-func FindAccount(dbif Dbif, id string) (*Account, error) {
-	tmp, err := strconv.ParseUint(id, ACCOUNT_ID_BASE, 64)
-	if err != nil {
-		return nil, err
-	}
+func FindAccount(dbif Dbif, id proto.UUID) (*Account, error) {
 	a := &Account{
 		proto.Model_Account {
-			Id: proto.UUID(tmp),
-		},
+			Id: id,
+		}, make(map[proto.UUID]*Persona),
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: proto
* [gotype] too many values in struct literal